service/src/main/kotlin/xyz/lebkuchenfm/domain/eventstream/EventStream.kt

```diff
@@ -1,5 +1,33 @@
 package xyz.lebkuchenfm.domain.eventstream
 
-interface EventStream {
-    suspend fun sendToEveryone(event: Event)
+import io.ktor.util.collections.ConcurrentMap
+import java.util.UUID
+
+abstract class EventStream<ConsumerT : EventStream.Consumer> {
+    protected val subscriptions: MutableMap<EventStreamConsumerId, ConsumerT> = ConcurrentMap()
```

Comment:
`streamConsumers`?

Reply:
I would prefer it as it is. You're subscribing to event stream, so this is a subscription list.